exercises/tournament/canonical-data.json

```diff
@@ -1,6 +1,6 @@
 {
   "exercise": "tournament",
-  "version": "1.1.0",
+  "version": "1.3.0",
```

Comment:
Removing the invalid input requirement was '1.2.0' adding the new initial tests made it '1.3.0', since '1.2.0' was not a separate PR and has never been released I can see an argument that we should stay on '1.2.0'.

I have a slight preference for '1.3.0' since it requires no work to change.

But if you think it should be '1.2.0' I'll change it.